eras/shelley/impl/src/Cardano/Ledger/Shelley/Gov.hs

```diff
@@ -76,10 +72,6 @@ class
   getProposedPPUpdates :: GovState era -> Maybe (ProposedPPUpdates era)
   getProposedPPUpdates _ = Nothing
 
-  -- | Returns `Nothing` for all era preceding Conway, otherwise returns the hash of the constitution
-  getConstitutionHash :: GovState era -> Maybe (SafeHash (EraCrypto era) ConstitutionData)
```

Comment:
I wonder why this was defined here, since it doesn't seem used in Shelley?  
But anyway, the lens version is more aligned with everything else :+1:

Reply:
Ah I see, because `queryConstitutionHash` is paramterized on Era, but now we've given it  `ConwayEraGov era` constraint, nevermind!

Reply:
It is used for the query in consensus, so we might still have to bring it back. I am trying to figure out right now in consensus if we can get away with removing it in Shelley.

Reply:
@Soupstraw We need to bring this function back, otherwise we can't implement the consensus ledger state query.

Reply:
It was kept here only because the previous functionality was implemented in Shelley :slightly_frowning_face: . I didn't have the time to introduce a `ConwayEraGov` before our demonstration. This is clean-up. Although, I remember we decided that we will break downstream quickly after and move it from Shelley to Conway-only.

Reply:
Replaced this function with `getConstitution`, since we do need something like that for consensus queries